src/components/PersonalPage/DevCard.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {Card, Image, ListGroup} from 'react-bootstrap';
+import "./DevCard.css"
+const swarmLogo = require('../../images/swarmLogoIcon.png');
+
+/*
+    User will be the prop for this component, passed in from the personal page component.
+    This will provide the user info, and provide necessary info the query the info the dev page. ie username/id
+*/
+
+
+class DevCard extends React.Component<any, any>{
```

Comment:
There needs to be Types associated with the components that are being imported. We don't want to have any specified anywhere in the application if we can avoid it.